lib/spack/spack/cmd/install.py

```diff
@@ -123,6 +177,12 @@ def setup_parser(subparser):
         default=False,
         help="only install package from binary mirrors",
     )
+    cache_group.add_argument(
+        "--use-buildcache",
+        dest="use_buildcache",
+        default="package:auto,dependencies:auto",
+        help="finer grain control over which packages to check mirrors for",
```

Comment:
Is this the right place to describe a little more to the user about the expected format?  The `default` value does provide an example, but the values `only`, `never`, and `auto` are hidden from the argument parser in the `parse_use_buildcache()` method.  It's also probably worth mentioning (here, if this is the right place) the shorthand versions allowed.

Reply:
Maybe also worth mentioning that `auto` means "fall back to the (maybe default) value of` either `--use-cache` or `--cache-only`.